docs/sources/flow/reference/components/local.file.md

```diff
@@ -94,3 +94,11 @@ component.
   timestamp, in Unix seconds, that the file was last sucessfully accessed.
 
 [secret]: ../secrets.md#is_secret-argument-in-components
+
+## Example
+
+```river
+local.file "my_file" {
+  filename = "path/to/my/file"
```

Comment:
PATH?

Reply:
Ah, this is supposed to be an example so it's not a placeholder. I'll make up a fake path.